docs/contributing/standards.md

```diff
@@ -6,38 +6,35 @@ sidebar_label: Standards
 
 # Standards
 
-The following standards are used by FAST-DNA contributors for consistency across this repository.
+Our contributors use these standards for consistency and quality in the repository.
 
-## Naming Conventions
+## Naming conventions
 
-* Avoid all abbreviations, except where common industry acronyms.
+* Avoid abbreviations unless they are common industry acronyms.
 * Use names that are clear and intuitive.
 * Use semantic names instead of presentational.
 
 ## Accessibility
 
-FAST-DNA works diligently from the ground up to enable accessibility in collaboration with legal, accessibility, and marketing teams. As an ecosystem to accelerate the building of website's and apps we ensure you have the tools to build accessible user exerpiences.
+Acknowledge accessibility from the start of each new feature in collaboration with the legal and accessibility. As a system for the building of websites and applications, we ensure you have the tools to create accessible user experiences.
 
 ### Animation
 
-* End users should have a mechanism to reduce or remove animations from their experience. The Safari devs have proposed a media query for reduced animation. ("prefers-reduced-motion").
+* Users should have a mechanism to reduce or remove animations from their experience. The Safari devs have proposed a media query for reduced animation. ("prefers-reduced-motion"). Animations longer than 3 seconds must provide a way to stop the motion.
 * Some users with vestibular disorders or other cognitive disabilities have problems with movement. A safe fallback for nearly all users is fade animations.
-* Flashing animations can cause seizures and should be avoided.
+* Avoid flashing animations can cause seizures or discomfort.
 
 ## JSS (JavaScript Style Sheets) usage
 
-JSS class name contracts follow a [BEM-like](http://getbem.com/naming/) convention.
+JSS class names follow a [BEM-like](http://getbem.com/naming/) convention but our implementation of BEM is slightly modified because dashes — the character used to delimit *modifiers* — cannot be used as JavaScript object keys without using string literals; use underscores instead. A single underscore separates an element from a block while two underscores separate a modifier from a block or element.
```

Comment:
The class names themselves are generated and in production will have nothing to do with the javascript key used to define the class. I think "contract" here is accurate but you could also use "interface" to describe the relationship.

Reply:
Fixed.